Dfe.Academies.External.Web/Pages/YourApplications.cshtml.cs

```diff
@@ -19,6 +19,8 @@ public HomeModel(IConversionApplicationRetrievalService conversionApplications,
         {
             _conversionApplications = conversionApplications;
             _logger = logger;
+            ExistingApplications = new();
```

Comment:
These properties can be initialised directly on line 12/13, then they don't need to be initialised here in the constructor. I think that's a bit neater? 

`        public List<ConversionApplication> ExistingApplications { get; set; } = new();
`